targets/TARGET_ARM_SSG/TARGET_MUSCA_B1/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 # Copyright (c) 2020-2021 ARM Limited. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-include(${MBED_PATH}/platform/FEATURE_EXPERIMENTAL_API/FEATURE_PSA/TARGET_TFM/TARGET_TFM_LATEST/scripts/mbed_set_post_build_tfm.cmake)
+include(${mbed-os_SOURCE_DIR}/platform/FEATURE_EXPERIMENTAL_API/FEATURE_PSA/TARGET_TFM/TARGET_TFM_LATEST/scripts/mbed_set_post_build_tfm.cmake)
```

Comment:
I'm wondering if we can use [CMAKE_MODULE_PATH](https://cmake.org/cmake/help/latest/variable/CMAKE_MODULE_PATH.html?highlight=cmake_module_path) to avoid hard coding all of the include paths. We could set the module directories in `mbed-os/CMakeLists` and then include modules with e.g `include(mbed_set_post_build_tfm)` in other files. The benefit of this approach, in my view, is if we ever move the location of these files we will have fewer changes to make.

Reply:
+1, I'll check how we can create the modules for our scripts